mscore/pianotools.cpp

```diff
@@ -288,7 +288,7 @@ void PianoKeyItem::paint(QPainter* p, const QStyleOptionGraphicsItem* /*o*/, QWi
       p->setRenderHint(QPainter::Antialiasing, true);
       p->setPen(QPen(Qt::black, .8));
       if (_pressed)
-            p->setBrush(QColor(255, 255, 128));
+            p->setBrush(QColor(30, 144, 255, 180));
```

Comment:
Ok, that's indeed the right spot.  But why harcoding this new value?
Could you use MScore::selectColor[0] and adjust the alpha if needed http://doc.qt.io/qt-4.8/qcolor.html#setAlpha
or better, add another color just like selectColor or frameColor in preferences.